src/test/java/com/github/pemistahl/lingua/api/LanguageDetectorBuilderJavaTest.java

```diff
@@ -20,14 +20,14 @@
 
 import static com.github.pemistahl.lingua.api.Language.ENGLISH;
 import static com.github.pemistahl.lingua.api.Language.FRENCH;
-import static com.github.pemistahl.lingua.api.Language.GERMAN;
+import static com.github.pemistahl.lingua.api.Language.CHINESE;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LanguageDetectorBuilderJavaTest {
 
     @Test
     public void assertThatLinguaCanBeUsedWithJava() {
-        final LanguageDetector detector = LanguageDetectorBuilder.fromLanguages(GERMAN, ENGLISH, FRENCH).build();
-        assertThat(detector.detectLanguageOf("groß")).isEqualTo(GERMAN);
+        final LanguageDetector detector = LanguageDetectorBuilder.fromLanguages(CHINESE, ENGLISH, FRENCH).build();
+        assertThat(detector.detectLanguageOf("上海大学是一个好大学  this is a test.")).isEqualTo(CHINESE);
```

Comment:
1. Please add a new test case instead of overwriting an existing one. 

2. Also, consider adding your new test case to the Kotlin tests, not to the Java tests.

Reply:
I will add test cases in new Kotlin file .